build.gradle

```diff
@@ -14,7 +14,7 @@ plugins {
     id "com.github.johnrengelman.shadow" version "4.0.3"
 }
 
-version = '2.6.0-SNAPSHOT'
+version = '2.7.0-SNAPSHOT'
```

Comment:
Hi,

Why is this a 2.6.x to 2.7.x update?

According to https://semver.org/:

> Given a version number MAJOR.MINOR.PATCH, increment the:
>
> 1. MAJOR version when you make incompatible API changes,
> 2. MINOR version when you add functionality in a backwards compatible manner, and
> 3. PATCH version when you make backwards compatible bug fixes.
>
>Additional labels for pre-release and build metadata are available as extensions to the MAJOR.MINOR.PATCH format.

Reply:
However I haven't followed semver in this project closely as well; it just surprised me to see this version jump.